docs-chef-io/content/inspec/dsl_inspec.md

```diff
@@ -249,6 +249,21 @@ control 'nutcracker-connect-redis-001' do
 end
 ```
 
+Here is another simple example, which checks for, if certain pip packages are installed, only if /root/.aws exists
```

Comment:
@unixengineer Hello! Please update your PR to include the following wording. Thank you for adding this example!

```suggestion
This example checks for if certain pip packages are installed, but only if '/root/.aws' exists:
```

Reply:
Will update the PR and add a signed by too

Reply:
The PR should have the updated changes now.